frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -84,7 +86,14 @@ export function RobotPage() {
                             battery={selectedRobot?.batteryLevel}
                             batteryStatus={BatteryStatus.Normal}
                         />
-                        <PressureStatusView itemSize={48} pressure={selectedRobot?.pressureLevel} />
+                        {selectedRobot && (
+                            <PressureStatusView
+                                itemSize={48}
+                                pressureInBar={selectedRobot?.pressureLevel}
+                                upperPressureWarningThreshold={selectedRobot?.model.upperPressureWarningThreshold}
+                                lowerPressureWarningThreshold={selectedRobot?.model.lowerPressureWarningThreshold}
```

Comment:
Why is selectedRobot still nullable? is it not sufficient to check whether it exists in line 89?